### DISCUSSION TITLE
Trying to create multiple objects at once, with different effects

### DISCUSSION THREAD

**Discussion by Ddupasquier** [matteobruni/tsparticles#4029] [General]
What I'm trying to achieve is 2 animations which happen on the same click event. A burst of circles, which I've got, as well as a single circle which expands to the full size of the screen and beyond, before disappearing. If this is achievable then I may add a third animations afterwards, but I'm just trying to start small. Having a heck of a time figuring out how to render 2 animations at the same time.

I also had an idea for the React.js side:
Maybe I just haven't been able to find it, but it would be awesome to have some kind of dynamic 'onClick' functionality. Like handleClick() {setBackground(newColor)}
Being able to alter state using a built-in click event would be AMAZING.

**Comment by matteobruni** [comment]
> 

You can use the `Container.addClickHandler` method to handle the click event in its dom element, that adds two emitters for two different particles set. (sample here: https://codesandbox.io/s/objective-colden-1rmyty?file=/src/App.tsx)
